docs/advanced/mapping.md

```diff
@@ -81,3 +107,23 @@ Note that if a `ledmap.json` file exists, the `2d-gaps.json` file will be ignore
 ```
 
 ![wiring diagram of the double ∞ shape mapping](mapping/mapping_infinity_shape.png)
+
+The following example demonstrates that when using sacrificial leds, you must include a mapping for all LEDs.
+
+Here, we have a serpentine of LEDs in 5 columns and 3 rows, and a sacrifical led between each row (in positions 5 and 11).
+Since the total number of LEDs in this system is 15 + 2 = 17, the `map` must also be 17 elements long.
+We fill those elements with -1.
+
+```json
+{
+  "map": [
+     0,  1,  2,  3,  4,
+    10,  9,  8,  7,  6,
+    12, 13, 14, 15, 16,
+
+    -1, -1
```

Comment:
* better to use "5, 11" instead of "-1,  -1" (see my previous comment)
* for best compatibility with older versions and other forks: don't use a blank line, and put "width" and "height"  before "map")